BUILD.gn

```diff
@@ -348,7 +348,7 @@ if (is_mac) {
     info_plist_target = ":brave_app_plist"
 
     extra_substitutions = [
-      "CHROMIUM_BUNDLE_ID=$chrome_mac_bundle_id",
+      "CHROMIUM_BUNDLE_ID=$chrome_mac_bundle_id.$brave_channel",
```

Comment:
Is this change needed w/o changes in below `BRANDING.dev`

Reply:
I'll go ahead and open an issue for this if that works, have some changes for this but needs more testing.